controllers/meshsync_controller.go

```diff
@@ -153,7 +153,7 @@ func (r *MeshSyncReconciler) reconcileMeshsync(ctx context.Context, enable bool,
 		if enable {
 			er := r.Update(ctx, object)
 			if er != nil {
-				return ctrl.Result{}, ErrUpdateMeshsync(er)
+				return ctrl.Result{}, ErrUpdateResource(er)
```

Comment:
Are we missing the complete error code definition?

Reply:
Probable cause and suggested remediation?

Reply:
Open for suggestions